examples/full-example/.mendelrc

```diff
@@ -210,4 +209,6 @@ env:
         outfile: server.js
         entries:
           - ./main.js
-        runtime: main # This
+        # If npm package has multiple entry files, this pulls only
+        # server side files defined in "main" field of package.json
+        runtime: main
```

Comment:
Frankly, this has to do with implementation of one of the outlet so it should go into respective outlets documentation. Thanks for the feedback. Maybe it is a better idea to write better documentation on auxiliary packages and put a link to those in this readme.